app/utils/sweats_message_handler.js

```diff
@@ -0,0 +1,141 @@
+import { DiscordRequest } from './discord_request.js';
+import { EventEmitter } from 'events';
+const green=0x66FF33
+const red=0xff6633
+
+export async function sweatsMessageHandler() {
+    try {
+        const channelId = process.env.CHANNEL_ID_CSWEATS;
+        const messageId = await sendMessage(channelId);
+        const intervalId = listenForReactions(channelId, messageId);
+        if (intervalId === null) {
+            return;
+        }
+        setTimeout(() => clearInterval(intervalId), 1000 * 60 * 60 * 24); // 24 hours
+    } catch (error) {
+        console.error(error);
+        throw new Error('Error handling sweats message');
+    }
+}
```

Comment:
(Suggestion: architecture) I think this function should move back to the `command_sweat_handler.js` and call `listenForReactions()` from this file (`message_utils.js`), etc.